submissions/april-1st/submission-april-1st.md

```diff
@@ -0,0 +1,208 @@
+# April 1st Mod Feature
+
+(v 1.2)
+
+
+
+## Working title: Funny Factorio Fan Formulations
+
+
+
+On the special occasion of Friday April 1st, we are proud to present some humorous picks from Factorio's vast modding scene. We will start with mods that only change visuals or audio to add some unique flavors to your eningeering experiences. Afterwards, we look at some crazy new weapons that spice up combat. Finally, we feature a mod with a level of transportation pizzaz that Willy Wonka himself would be proud of.
+
+
+
+### Sights and Sounds
+
+The mod portal features quite a few mods small and large that offer changes to visuals and/or audio. Here is a fun selection:
+
+
+
+**Emotes and Speech Bubbles** 
+
+by [Kiplacon](https://mods.factorio.com/user/Kiplacon)
+
+[This handy mod](https://mods.factorio.com/mod/EmotesAndSpeechBubbles) makes multiplayer interactions a lot livelier, especially when you can't use a mic. Your text in chat also appears over your head. This is already more immersive but the best part is the emotes it provides. The emote menu is opened by clicking the middle mouse button (and customized by right-clicking them). The mod helps enhance your fun and funny moments: There are emotes for partying, coordinating, and of course pressing "F" or declaring "rekt!" when your friend gets hit by a train. 
```

Comment:
I think here you could expand slightly that it's easier to display emotions, make jokes, or even make fun of your friends, or something like that :)